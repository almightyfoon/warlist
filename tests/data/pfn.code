c6501b_-0J5pc55X5F3W7mfj733G3H8M___-0Jb_94949293939393j63WfCiriq8Mk8hX
