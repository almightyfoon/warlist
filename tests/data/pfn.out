https://conflictchamber.com/#c6501b_-0J5pc55X5F3W7mfj733G3H8M___-0Jb_94949293939393j63WfCiriq8Mk8hX

Mercenary Army - 75 / 75 points
[Theme] The Irregulars

[Magnus 2] Magnus the Warlord [+28]
 - Galleon [39]
 - Mangler [15]
 - Renegade [10]
Eiryss, Mage Hunter of Ios [0(7)]
Kell Bailoch [0(5)]
Savio Montero Acosta [0(6)]
Croe's Cutthroats (max) [16]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [0(5)]
Lady Aiyana & Master Holt [8]

Mercenary Army - 75 / 75 points
[Theme] The Irregulars

[Ossrum 1] General Ossrum [+28]
 - Ghordson Driller [10]
 - Ghordson Driller [10]
 - Grundback Blaster [6]
 - Grundback Gunner [6]
 - Grundback Gunner [6]
 - Grundback Gunner [6]
 - Grundback Gunner [6]
Eilish Garrity, the Occultist [0(5)]
Eiryss, Mage Hunter of Ios [0(7)]
Hutchuk, Ogrun Bounty Hunter [0(6)]
Horgenhold Artillery Corps [6]
Horgenhold Forge Guard (max) [16]
Lady Aiyana & Master Holt [8]
Lynus Wesselbaum & Edrea Lloryrr [5]
Hammerfall Siege Crawler [18]
